app/Modules/Recruitment/Http/Requests/JobAdvertRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Modules\Recruitment\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
```

Comment:
You don't need Rule and Route here. Make sure you only include the classes you use.